support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -432,7 +448,7 @@ const paymentButtonStyle = {
 		theme: 'dark',
 		height: '42px',
 	},
-};
+} as const;
```

Comment:
This was pretty handy! Without it it's interpreted as 

```ts
{
  theme: string;
  height: string;
}
```

but adding as `const` makes e.g the type of `theme` as the literal string `dark`. The stripe library required the `theme` to be one of a union of several strings